### PR TITLE
[mieb] add siglip, cohere multimodal & some fixes for final run

### PR DESCRIPTION
**Pull request by gowitheflow-1998**
1. added Siglip; tested on multilingual retrieval.
2. added Cohere multimodal model. Cohere API currently only allows 40 images/min so only tested on a image clustering task with time.sleep(1.5) per image. Added a `-v` in model name for now to distinguish with the text-only wrapper. Can merge with the text-only one in the future as they are using the same model.
3. fixed task type error of a few remaining datasets.
4. changed image clustering main metric to nmi & delete accuracy (was wrong and replaced by `clustering accuracy`)
5. updated `run_mieb.py` cc @Muennighoff 

## Checklist

- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`.

### CODE REVIEW

**Review by imenelydiaker — APPROVED** [review]
All good for Clustering and all corrected tasks! 🚀

File: mteb/evaluation/evaluators/Image/ClusteringEvaluator.py [hunk 1]
```diff
@@ -70,7 +68,6 @@ def __call__(self, model: Encoder, *, encode_kwargs: dict[str, Any] = {}):
 
         return {
             "v_measure": v_measure,
-            "accuracy": accuracy,
```
Comment (KennethEnevoldsen) [inline mteb/evaluation/evaluators/Image/ClusteringEvaluator.py:73]: > changed image clustering main metric to nmi & delete accuracy

Any reason to go for nmi over the v-measure?

Comment (gowitheflow-1998) [reply mteb/evaluation/evaluators/Image/ClusteringEvaluator.py:73]: They actually always produce the same values, while NMI is a more known metric in image clustering I think! (e.g., [benchmark](https://paperswithcode.com/sota/image-clustering-on-imagenet) [benchmark](https://paperswithcode.com/sota/image-clustering-on-tiny-imagenet) [paper](https://openaccess.thecvf.com/content_ICCV_2017/papers/Chang_Deep_Adaptive_Image_ICCV_2017_paper.pdf) [paper](https://arxiv.org/pdf/2402.10093v2) [paper](https://arxiv.org/pdf/2303.17896v2))

**Review by isaac-chung — APPROVED** [review]
Looks great 🚀 one non-blocking suggestion

File: mteb/evaluation/evaluators/Image/ClusteringEvaluator.py (hunk 1 above)
Comment (KennethEnevoldsen) [reply mteb/evaluation/evaluators/Image/ClusteringEvaluator.py:73]: Thanks! Then there is no worries

File: mteb/models/cohere_v.py [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+from torchvision import transforms
+from tqdm import tqdm
+import os
+import io
+import base64
+import mteb
+import time
+from mteb.model_meta import ModelMeta
+
+api_key = os.getenv("API_KEY")
```
[the remark this answers is not in the document]
Comment (gowitheflow-1998) [reply mteb/models/cohere_v.py]: sure, fixed!

**Pull request merged**